scripts/update-3rd-party.ts

```diff
@@ -38,13 +34,15 @@ const cleanSchemaObject = (obj: any) => {
 const inlineRemoteRefs = async (json: any): Promise<void> => {
     for (const entry of json.allOf) {
         if (entry.$ref && entry.$ref.startsWith('https://')) {
-            const res = await request({ url: entry.$ref }) as req.Response;
+            const res = await fetch(entry.$ref);
 
-            if (res.body.message) {
-                throw new Error(res.body.message);
-            }
-
-            const refJson = JSON.parse(res.body);
+            /**
+             * [vidorteg] validate if this is needed
+             * if (res.body.message) {
+             *      throw new Error(res.body.message);
+             * }
+             */
```

Comment:
@anton do you know what this script is used for or how to test this change?

Reply:
I believe this is for when we are pulling schema definitions from different places (can't remember exactly which ones).
This seems to be a safeguard for when the server is down or similar so I'd keep it because it has probably happened in the past.